### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by Reaster0** [illixion/vscode-vibrancy-continued#27]
and here we are again, searching for a way of using vibrancy in GNU/linux os

**Comment by PoSayDone** [comment]
If u are arch linux user you can use this - https://aur.archlinux.org/packages/code-transparent
There is also pre-builded package in archlinuxcn repo.

**Comment by Reaster0** [comment]
unfortunately i'm on ubuntu

**Comment by illixion** [comment]
For what it's worth, [GlassIt-VSC](https://marketplace.visualstudio.com/items?itemName=s-nlf-fh.glassit) should work on Linux, though it makes the whole window semi-transparent.

**Comment by microhobby** [comment]
I was using `glasstron` for my own VS Code build, and it's work on Windows and Linux (does not know about Mac OS).

**Comment by Reaster0** [comment]
thanks for your answer, for the next ones that will read us, i can't find `glasstron` but i've found `Vibracy Continued `on the vscode store, it work for windows and macos (but apparently not linux)

**Comment by illixion** [comment]
> thanks for your answer, for the next ones that will read us, i can't find `glasstron` but i've found `Vibracy Continued `on the vscode store, it work for windows and macos (but apparently not linux)

Yeah, this is the repo for Vibrancy Continued. [Glasstron](https://github.com/NyaomiDEV/Glasstron) seems to be a module you can include in your own VSCode build, but it says that it's been deprecated, so I'm not sure if it'll continue working for long.

**Comment by Visual-Dawg** [comment]
https://aur.archlinux.org/packages/code-translucent
This build allows you to set the background as transparent. It works nicely.
An extension would still be nicer though

**Comment by kevinnkansah** [comment]
Still no Linux support???

**Comment by Visual-Dawg** [comment]
@kevinnkansah Just use https://marketplace.visualstudio.com/items?itemName=drcika.apc-extension
It supports Linux and is actually way nicer

**Comment by heysokam** [comment]
> @kevinnkansah Just use https://marketplace.visualstudio.com/items?itemName=drcika.apc-extension It supports Linux and is actually way nicer

> [...] _just use_ [...]

I'm afraid there is no `just` in this process.
The extension installs, and seems to work, but there is no explicit option for window opacity, and documentation is atrocious :(
Can you explain how to enable window transparency with this extension you linked?

**Comment by kevinnkansah** [comment]
Linux support after 3 years, this year must be the year of the Linux desktop.📈
